packages/mobile/ios/celo/Info.plist

```diff
@@ -49,9 +49,11 @@
 		</dict>
 	</dict>
 	<key>NSCameraUsageDescription</key>
-	<string>This is so you can scan QR codes.</string>
+	<string>Celo Wallet needs camera access to scan QR codes</string>
+	<key>NSPhotoLibraryAddUsageDescription</key>
+	<string>Celo Wallet needs access to save your QR code to your photos</string>
```

Comment:
nit: isn't having 'Celo Wallet' in the description a bit redundant? given "Celo" is already in the title.
![74013948-e4704200-498d-11ea-81ba-95da30eb39b5_censored](https://user-images.githubusercontent.com/57791/74014254-927bec00-498e-11ea-92d8-975489e093cb.jpg)

We should keep the "." at the end of the sentence.

Reply:
I'll add back the `.` but I think having Celo Wallet in the description looks more professional. I saw other examples online of people doing that.